tests/draft2019-09/const.json

```diff
@@ -125,6 +125,90 @@
             }
         ]
     },
+    {
+        "description": "const with [false] does not match [0]",
+        "schema": {"const": [false]},
```

Comment:
Should these also be added to `enum`? Likely there's the same test gap there as here since these are new extra cases.

Reply:
(And that would apply all the way back to old drafts if so)